common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -198,6 +220,20 @@ def update_db(self, pallet_info, URL):
 			# Re-added the pallet. Update the URL.
 			self.db.execute('UPDATE rolls SET url=%s WHERE id=%s', (URL, rows[0][0]))
 
+	def perform_checksum(self, pallet_arg, iso_name):
+		if pallet_arg in self.checksum:
+			checksum_type, checksum_value = self.checksum[pallet_arg]
+			self.notify(f'Checking {iso_name} matches {checksum_type}={checksum_value}...')
+			checksum_input = f'{checksum_value} {iso_name}\n'
+			proc = self._exec(
+				cmd = f'{checksum_type}sum -c',
+				shlexsplit=True,
+				input=checksum_input
+			)
```

Comment:
Don't need the `checksum_input` variable. Just put its value directly in the _exec call.